codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/protocol/HttpBindingProtocolGenerator.kt

```diff
@@ -266,7 +266,11 @@ abstract class HttpBindingProtocolGenerator : ProtocolGenerator {
             writer.write("val payload = #T(context, input)", opBodySerializerFn)
             writer.write("builder.body = #T.fromBytes(payload)", RuntimeTypes.Http.HttpBody)
         }
+        renderContentTypeHeader(ctx, op, writer)
+    }
 
+    protected open fun renderContentTypeHeader(ctx: ProtocolGenerator.GenerationContext, op: OperationShape, writer: KotlinWriter) {
+        val resolver = getProtocolHttpBindingResolver(ctx.model, ctx.service)
```

Comment:
**Nit**: Pass `resolver` through to `renderContentTypeHeader`.

Reply:
We don't do that elsewhere FWIW (e.g.`renderUri` below)